Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -3263,10 +3284,51 @@ void webkit_settings_set_enable_media_stream(WebKitSettings* settings, gboolean
 
     priv->preferences->setMediaDevicesEnabled(enabled);
     priv->preferences->setMediaStreamEnabled(enabled);
-    priv->preferences->setPeerConnectionEnabled(enabled);
     g_object_notify_by_pspec(G_OBJECT(settings), sObjProperties[PROP_ENABLE_MEDIA_STREAM]);
 }
 
+/**
+ * webkit_settings_get_enable_webrtc:
+ * @settings: a #WebKitSettings
+ *
+ * Get the #WebKitSettings:enable-webrtc property.
```

Comment:
Use `[property@Settings:enable-webrtc]` for [linking things with gi-docgen](https://gnome.pages.gitlab.gnome.org/gi-docgen/linking.html).

Reply:
It's because we haven't been converting everything in bulk—so far we did a rather minimal conversion; I have plans to work on this kind of fixups in the next months 😄